contracts/crowdsale/validation/TimedCrowdsale.sol

```diff
@@ -19,7 +19,7 @@ contract TimedCrowdsale is Crowdsale {
    */
   modifier onlyWhileOpen {
     // solium-disable-next-line security/no-block-members
```

Comment:
We can remove this Solium annotation because we moved the expression to a function.

Reply:
Removed.